packages/playwright-mcp/src/resources/resources.ts

```diff
@@ -14,22 +14,22 @@
  * limitations under the License.
  */
 
-import { Server } from './server';
-import { navigate, wait, pressKey } from '../tools/common';
-import { screenshot, moveMouse, click, drag, type } from '../tools/screenshot';
+import type { Resource } from './resource';
 
-const server = new Server({
-  name: 'Playwright screenshot-based browser server',
-  version: '0.0.1',
-  tools: [
-    navigate,
-    screenshot,
-    moveMouse,
-    click,
-    drag,
-    type,
-    pressKey,
-    wait,
-  ]
-});
-server.start();
+export const pdf: Resource = {
+  schema: {
+    uri: 'file:///playwright/page.pdf',
```

Comment:
is 'file:' uri expected?

Reply:
no, that's exactly what the require in the spec